public/views/adminRights.html

```diff
@@ -0,0 +1,27 @@
+<h2>Configuration des droits des usagers</h2>
+<h4>Filtres</h4>
+<div class="row">
+    <div class="col-md-3">
+        Nom: <input type="text" class="form-control" ng-model="nameFilter" ng-change="filter('cipFilter', 'nameFilter')">
+    </div>
+    <div class="col-md-1" ng-repeat="right in rights">
```

Comment:
The upper checkboxes are very useful IMO. When we will have 300+ users it will be super useful to filter all the users by their permissions. e.g: Only keep the admins or only the permies. I think it will save the users quite a bit of time!